pkg/flow/internal/controller/loader.go

```diff
@@ -109,6 +109,14 @@ func NewLoader(opts LoaderOptions) *Loader {
 	return l
 }
 
+// LoadOptions are options that can be provided when loading a new River config.
+type LoadOptions struct {
```

Comment:
nit: this is passed to Apply, so ApplyOptions (or LoaderApplyOptions) would be a more descriptive name.